WORKSPACE

```diff
@@ -26,6 +26,13 @@ http_archive(
     ],
 )
 
+http_archive(
```

Comment:
What introduced the need for this dependency? If it is actually required, lets move it to proto_deps.bzl and follow the pattern we're using there.

Reply:
Removed this dependency.